src/main/java/com/targomo/client/api/quality/criterion/StatisticsOnEnclosingCellCriterionDefinition.java

```diff
@@ -0,0 +1,38 @@
+package com.targomo.client.api.quality.criterion;
+
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.fasterxml.jackson.databind.annotation.JsonDeserialize;
+import com.fasterxml.jackson.databind.annotation.JsonPOJOBuilder;
+import lombok.EqualsAndHashCode;
+import lombok.Getter;
+import lombok.Setter;
+import lombok.ToString;
+import lombok.experimental.SuperBuilder;
+
+import javax.validation.constraints.NotEmpty;
+import java.util.List;
+
+@EqualsAndHashCode(callSuper = true)
+@SuperBuilder(toBuilder = true)
+@ToString(callSuper = true)
+@Getter
+@JsonDeserialize(builder = StatisticsOnEnclosingCellCriterionDefinition.StatisticsOnEnclosingCellCriterionDefinitionBuilderImpl.class)
+@JsonInclude(JsonInclude.Include.NON_EMPTY)
+public class StatisticsOnEnclosingCellCriterionDefinition extends CriterionDefinition implements StatisticsCriterionDefinition{
+
+    private final Integer statisticGroupId;
+    private final Integer statisticCollectionId;
+
+    @NotEmpty
+    private final List<Short> statisticsIds;
+    private final List<Short> referenceStatisticsIds;
+
+    @Setter
+    private String statisticsServiceUrl;
+
+    private final String coreServiceUrl;
```

Comment:
you can always set it when you build it. But for this criteria we don't need to update it after the initialization (because no need for a proxy url replacement etc). This field can be final without problem